clients/vault/src/lib.rs

```diff
@@ -21,13 +20,13 @@ mod system;
 
 mod issue;
 pub mod oracle;
+mod requests;
```

Comment:
`mod requests` replaces `mod execution`. It splits the old execution.rs into 3: 
* execution.rs
* helper.rs
* structs.rs